src/sentry/features/flagpole_utils/sentry_flagpole_context.py

```diff
@@ -0,0 +1,102 @@
+from dataclasses import dataclass
```

Comment:
Nit: You could call the module `sentry.features.flagpole`, or even have this in `sentry.features.flagpole_context` :shrug: